modules/nf-core/universc/count/main.nf

```diff
@@ -0,0 +1,59 @@
+process UNIVERSC_COUNT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using the Cell Ranger tool. Please use docker or singularity containers."
+        conda (params.enable_conda ? "hcc::cellranger=3.0.2" : null)
+    }
+    container "nfcore/universc:1.2.4"
+    if (workflow.containerEngine == 'docker'){
+        containerOptions = "--user root"
+    }
+    if (workflow.containerEngine == 'singularity'){
+        containerOptions = "--writable"
+    }
+
+    input:
+    tuple val(meta), path(reads)
+    path  reference
+
+    output:
+    tuple val(meta), path("sample-${meta.id}/outs/*"), emit: outs
+    path "versions.yml"                              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def sample_arg = meta.samples.unique().join(",")
+    def reference_name = reference.name
+    """
+    cellranger \\
```

Comment:
There is already a module for cellranger count, is it not possible to use that one here? https://github.com/nf-core/modules/tree/master/modules%2Fnf-core%2Fcellranger%2Fcount

Reply:
@ggabernet Thanks for taking time to review! As explained above this runs a different version of Cell Ranger (v3.0.2) on an open-source license (not subject to the 10X Genomics EULA). I recommend running it via the `universc/launch` submodule for compatibility with technologies other than 10X Genomics Chromium.

The `universc/mkref` module is therefore needed to generate a compatible STAR index with the matching dependency versions. You're right that running `cellranger count` or `cellranger mkfastq` in older version of Cell Ranger are not necessary for this. I think unit testing the `cellranger count` call with `universc/count` separately from `universc/launch` is still important. 

> same for cellranger mkfastq

As raised below `universc/mkfastq` for demultiplexing is specific to 10X Genomics indexes. There is not much benefit to running it in the open-source version (although I have checked it works). I was considering to remove this one already actually.

Reply:
The submodules `universc/mkfastq` and `universc/count` have been removed as these can already by run via the cellranger module. Added functionality is run in `universc/launch` with references generated by compatible versions.

Changes shown here: https://github.com/nf-core/modules/pull/1706/commits/ec26f7ede4c9f58c2f65daac8e701716a6170dfb

Sorry for confusing merge commits from the wrong account.